lib/services/security/service.js

```diff
@@ -229,7 +229,9 @@ function initialize(config, callback) {
     .then(() => {
       callback();

```

Comment:
please remove the .then(callback).catch(callback) pattern. Probably best to make this a callbackified async function

Reply:
this has been done